testing/regress/ecl/signingFail.ecl

```diff
@@ -0,0 +1,41 @@
+//nokey
+//version version=0
+//version version=1
+//version version=2
+//version version=3
+//version version=4
+
+import  ^ as root;
+v := #IFDEFINED(root.version, 1);
+
+IMPORT Std;
+
+#IF (v = 0)
+    Std.Outer_Macro1US(4);
```

Comment:
These macros are not defined, and should not be in Std.  I suspect it will require a signed file checked into this directory.

Reply:
Added macros to testing/regress/ecl/signing